src/lib/prepareOas.ts

```diff
@@ -40,7 +40,17 @@ export default async function prepareOas(path: string, command: 'openapi' | 'val
 
     const fileFindingSpinner = ora({ text: 'Attempting to locate API definitions...', ...oraOptions() }).start();
 
-    const action = command === 'openapi' ? 'upload' : 'validate';
+    let action;
```

Comment:
TS noob question but is there any benefit to a stricter type here?

```suggestion
    let action: string;
```
or maybe even this:
```suggestion
    let action: 'upload' | 'reduce' | 'validate';
```

Reply:
It's set up as an `any` but because we're filling it with strings when it's used its implied type ends up being a string:

![Screen Shot 2022-08-19 at 9 32 30 AM](https://user-images.githubusercontent.com/33762/185665496-c71cdb89-c35a-4895-89c4-bd094c26acf9.png)

I like the more specific type union option though.